src/App.js

```diff
@@ -1,23 +1,46 @@
-import logo from './logo.svg';
 import './App.css';
+import React, { useState } from 'react';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
+import Search from './components/Search';
+
 
 function App() {
+  const [dishes, setDishes] = useState(foods)
+  const createFood = (newDishes)=>{
+    const edit = [newDishes, ...dishes]
+    setDishes(edit)
+  }
+const searchFood = (searchQuery)=>{
+  const filteredFood = dishes.filter((food) => 
+  food.name.toLowerCase().includes(searchQuery.toLowerCase())
+  );
+  setDishes(filteredFood)
```

Comment:
The `dishes` will be the original array that will always contain every food. So here you want to update the `setShowFoods` instead of the `setDishes` like this:
`setShowFoods(filteredFood)`